benchmark/stripe82/validate_on_stripe82.jl

```diff
@@ -65,25 +69,38 @@ where
 """
 const truthfile = joinpath(datadir, "coadd_for_4263_5_119.fit")
 
+const valid_args = Set(["--score-only", "--joint-infer", "--fft"])
 
-if length(ARGS) > 1 || (length(ARGS) == 1 && ARGS[1] != "--score-only")
-    println("usage: validate_on_stripe82.jl [--score-only]")
+if !(ARGS ⊆ valid_args)
+    args_str = join(["[$va]" for va in valid_args],  " ")
+    println("usage: validate_on_stripe82.jl $args_str")
 else
     # By default this script both infers all the parameters and scores them,
     # but because inference is computationally intensive, whereas scoring isn't,
     # the user gets the option to just run the scoring mode. Running a scoring
     # alone would primarily be useful for debugging.
-    if length(ARGS) == 0 || ARGS[1] != "--score-only"
-        # adding `; objid=...` to the call limits infer_field to that source.
-        # Potentially useful for debugging.
-        @time infer_field(rcf, datadir, outdir, joint_infer=true)
+    if !("--score-only" in ARGS)
+        wrap_joint(cnti...) = one_node_joint_infer(cnti...;
+                                                   termination_percent=0.9)
+        source_callback = "--fft" in ARGS ? infer_source_fft : infer_source
```

Comment:
@rgiordan  -- You might do the same kind of thing in the galsim benchmarks with a `--fft` flag.